docs/source/user-guide/tuning.md

```diff
@@ -21,6 +21,17 @@ under the License.
 
 Comet provides some tuning options to help you get the best performance from your queries.
 
+## Memory Tuning
+
+Comet currently doesn't share the memory allocation from Spark but owns its own memory allocation.
+That's said, Comet requires additional memory to be allocated. Comet provides some memory related configurations to help you tune the memory usage.
+
+By default, the amount of memory is `spark.comet.memory.overhead.factor` * `spark.executor.memory`.
```

Comment:
I think we need to specify the precedence as well.
What if both `spark.comet.memory.overhead.factor` and `spark.comet.memoryOverhead` are set ?

Reply:
`spark.comet.memoryOverhead` will be used. Let me add it.